packages/building-rollup/modern-config.js

```diff
@@ -14,6 +14,8 @@ const getWorkboxConfig = require('@open-wc/building-utils/get-workbox-config');
 
 const production = !process.env.ROLLUP_WATCH;
 
+const DEFAULT_DEDUPE_MODULES = ['lit-html'];
```

Comment:
I think we should add `lit-element` here as well.

Reply:
`rollup-plugin-node-resolve`'s `dedupe` option can take as its value either an array of package names, or a function of `importee -> boolean`

So we could pass `() => true` to dedupe everything, meaning our option's type could be `Boolean | Function | 'all' | string[]`, and in case of 'all', we'd pass that constant true func.

Reply:
Yes, I see now that lit-element uses `instanceof` for `css` tagged templates
https://github.com/Polymer/lit-element/blob/f552fc47b12bf3e272aaeafe03d9d73bc4fff1f5/src/lib/css-tag.ts#L64